core/src/display_object/movie_clip.rs

```diff
@@ -1836,7 +1836,14 @@ impl<'gc> MovieClip<'gc> {
             let result: Result<(), Avm2Error> = constr_thing();
 
             if let Err(e) = result {
-                log::error!("Got {} when constructing AVM2 side of movie clip", e);
+                log::error!(
+                    "Got {} when constructing AVM2 side of movie clip of type {}",
+                    e,
+                    class_object
+                        .try_inner_class_definition()
+                        .map(|c| c.read().name().to_qualified_name(context.gc_context))
+                        .unwrap_or_else(|_| "[BorrowError!]".into())
```

Comment:
why `BorrowError`?

Reply:
We're using `try_inner_class_definition` which returns a `BorrowError` if the class we're trying to read is being mutated on the same stack. Otherwise, we would panic when reading the name and lose the rest of the error message in the process.

I don't know if such a situation could actually be triggered by user code.

Reply:
Oh, yeah, I just read it now. OK, so that's the "we don't know if this can ever happen but this method exists just to make sure we don't ever panic from here", right? Looks OK then.